platform/admin/core/admin-core-logviewer/src/main/webapp/js/components/log-entry/log-entry.js

```diff
@@ -46,20 +49,37 @@ export default ({ entry, marks }) => {
     }
   }
 
+  const expandEntry = () => {
+    dispatch(actions.expandEntry(entry.hash))
+  }
+
+  const getMessageClasses = () => {
+    return (entry.hash === expandedHash) ? 'rowData messageExpanded' : 'rowData message'
+  }
+
+  const getRowClasses = () => {
+    if (entry.hash === expandedHash) {
+      return 'rowAnimation selectedRow'
+    } else {
+      return 'rowAnimation'
+    }
+  }
+
   return (
-    <tr className='rowAnimation'>
-      <td className='row' style={{ background: levels(entry.level) }} width={175}>
+    <tr onClick={expandEntry} className={getRowClasses()} style={{ background: levels(entry.level) }}>
```

Comment:
Fixed. I modified the levels.js to return class names instead of colors directly, and added classes to log-entry.less to set the background colors.